### DISCUSSION TITLE
Let's meet at NAB2023

### DISCUSSION THREAD

**Discussion by getroot** [OvenMediaLabs/OvenMediaEngine#1065] [Announcement]
The OvenMediaEngine development team will be exhibiting at the 2023 NAB exhibition.

Contributors, please visit us when you come to NAB. If you let us know your visit schedule in the comments here or at support@airensoft.com in advance, I will arrange a time to meet you and prepare a coffee.

For more details, please refer to the blog below.
https://medium.com/@AirenSoft/airensoft-participates-in-the-2023-nab-show-8fcf17f0e87b

**Comment by bchah** [comment]
See you there 👍

**Comment by stevespaw** [comment]
I will be there!

**Comment by alexthedamager** [comment]
See you there!

**Comment by getroot** [comment]
Dear all,

I want to express my sincere gratitude to each and every one of you who took the time to attend the NAB exhibition where my team and I showcased OvenMediaEngine. It was an absolute pleasure to finally meet in person with members of our community whom we had only previously known online.

Your advice and support mean the world to us, and we're committed to incorporating your suggestions to make OvenMediaEngine even better. I'm already looking forward to our next opportunity to connect in person.

I must also say that there is something truly special about offline meetings that cannot be replicated online. Therefore, I will continue to use the financial support OvenMediaEngine receives to participate in exhibitions and conferences. I believe these events provide invaluable opportunities to connect with our community, share knowledge, learn from others, and showcase the latest developments in our technology.

And don't worry, I promise to work on improving my English skills for next time, so we can have even more productive conversations. (I felt this more desperately this time. It would be great if English was as easy as C++... :( )

Thank you again for your support and encouragement!

Warm regards,
Jeheon
